clang/test/Driver/sycl-no-gpu-rdc-opaque-pointers.cpp

```diff
@@ -0,0 +1,27 @@
+///
+/// Checks that the SYCL libraries modules are first linked with opaque-pointers, 
+/// when no GPU Relocatable Device Code mode is used.
+///
+
+#include <sycl/sycl.hpp>
+
+int main() {
+  sycl::queue q;
+  auto ptr = sycl::malloc_shared<int>(1, q);
+  q.single_task([=]() {
+	  ptr[0] = 1;
+  });
+  return 0;
+}
```

Comment:
There isn't any actual compilation occurring - the presence of any source code is not needed.

Reply:
Correct yeah. Thank you! Updated now.